theme/views.py

```diff
@@ -654,6 +657,11 @@ def email_verify(request, new_email, uidb36=None, token=None):
     if user is not None:
         user.email = new_email
         user.save()
+        try:
+            keycloak_id = KEYCLOAK_ADMIN.get_user_id(user.get_username())
+            KEYCLOAK_ADMIN.update_user(keycloak_id, payload={"email": new_email})
```

Comment:
okay, I see this now
https://python-keycloak.readthedocs.io/en/latest/modules/admin.html#update-user
looks good.